superset-frontend/src/views/CRUD/welcome/EmptyState.tsx

```diff
@@ -103,7 +116,7 @@ export default function EmptyState({ tableName, tab }: EmptyStateProps) {
               <Button
                 buttonStyle="primary"
                 onClick={() => {
-                  window.location = mineRedirects[tableName];
+                  window.location.href = mineRedirects[tableName];
```

Comment:
Since `mineRedirects[tableName]` is `string`, we need to assign it to `location.href` as stated here: https://stackoverflow.com/questions/13106950/set-window-location-with-typescript/13106955